eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -287,15 +286,16 @@ jobs:
             coverageThreshold: ${{ parameters.LineCoverageTarget }}
           condition: and(succeededOrFailed(), eq(variables['CODE_COVERAGE'], 'enabled'), eq(variables['Skip.LineCoverageEnforcement'], ''))
 
-        - task: mspremier.BuildQualityChecks.QualityChecks-task.BuildQualityChecks@8
-          displayName: Check branch coverage
-          inputs:
-            checkCoverage: true
-            coverageFailOption: fixed
-            coverageType: branches
-            # Minimum baseline for branch coverage
-            coverageThreshold: ${{ parameters.BranchCoverageTarget }}
-          condition: and(succeededOrFailed(), eq(variables['CODE_COVERAGE'], 'enabled'), eq(variables['Skip.BranchCoverageEnforcement'], ''))
+#  Disabling branch coverage check until newer tasks can support it see issue https://github.com/microsoft/azure-pipelines-tasks/issues/19669 tracking that request.
```

Comment:
@RickWinter just wanted to call this to your attention to ensure your team is OK with disabling this branch coverage check.

Reply:
Honestly? I think you'll find that the team *welcomes* the removal of the branch check, because it's fundamentally unreliable.

Reply:
Sounds good we might add it back in the future if the support comes back.